src/ctapipe/fitting.py

```diff
@@ -2,6 +2,7 @@
 from numba import njit
 
 EPS = 2 * np.finfo(np.float64).eps
+FIT_RNG = np.random.default_rng(0)
```

Comment:
This is not an example, right? Why the initialization is done with a fixed seed?

Reply:
To make it reproducible by default.